### DISCUSSION TITLE
Feature request: option or event that only triggers when multiple workflows are completed

### DISCUSSION THREAD

**Discussion by lwhiteley** [community/community#26415] [Actions]
<p>Currently I am using the the very helpful <code>worflow_run</code> event</p>
<pre><code class="lang-auto">on:
  workflow_run:
    workflows: ["workflow-1", "workflow-2"]
    types: 
      - completed
</code></pre>
<p>While this is cool, It would also be helpful to have another event or option that triggers when all listed workflows are completed.</p>
<p>Maybe this means that a new event may need to be created as there would need to be an array of workflow payloads.</p>
<ul>
<li>Is this already possible and im missing it?</li>
<li>or is this something planned or won’t do?</li>
</ul>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/lwhiteley">@lwhiteley</a>,</p>
<blockquote>
<p>Is this already possible and im missing it?</p>
</blockquote>
<p>No, currently we do not have this feature to do that.</p>
<blockquote>
<p>is this something planned or won’t do?</p>
</blockquote>
<p>You can follow the “<a href="https://github.com/github/roadmap/issues?q=is%3Aopen+is%3Aissue+label%3Aactions" rel="noopener nofollow ugc"><strong>GitHub public roadmap</strong></a>” repository where you can learn about what new features the appropriate engineering teams are working on, what stage these features in, and when the teams expect to bring them to users.<br>
I have checked on this repository, the feature you require seems has not been on the plan currently.</p>
<p>In addition, I recommend that you also can directly share a feature request in the <a href="https://support.github.com/contact/feedback" rel="noopener nofollow ugc"><strong>Feedback form for GitHub</strong></a>. That will allow you to directly interact with the appropriate engineering team, and make it more convenient for the engineering team to collect and categorize your suggestions.</p>
